src/Search.php

```diff
@@ -3427,8 +3427,10 @@ public static function displaySearchoptionValue($request = [])
 
        // Default case : text field
         if (!$display) {
+            global $DB;
+            $pattern = $DB->getFieldPattern($searchopt['table'], $searchopt['field'], false);
```

Comment:
$searchopt can be an empty array here at least when there is an Items Seen criteria since it isn't actually a search option.